sqlitedb/queries.md

```diff
@@ -11,14 +11,14 @@
 ```Shell
    .open dcrdata.sqlt.db
 
-    CREATE TABLE test_dcrdata_block_summary AS SELECT * FROM dcrdata_block_summary WHERE height > 131800 LIMIT 200;
-    CREATE TABLE test_dcrdata_stakeinfo_extended AS SELECT * FROM dcrdata_stakeinfo_extended WHERE height > 131800 LIMIT 200;
+   CREATE TABLE test_dcrdata_block_summary AS SELECT * FROM dcrdata_block_summary WHERE height < 200;
+   CREATE TABLE test_dcrdata_stakeinfo_extended AS SELECT * FROM dcrdata_stakeinfo_extended WHERE height < 200;
 ```
 
 - Create dump files with the newly created tables' data and dump data into `data_1.sql` and `data_2.sql` as shown.
 ```Shell
     .output data_1.sql
-    .dump test_dcrdata_block_summary;
+    .dump test_dcrdata_block_summary
```

Comment:
I may have an outdated version of `sqlite3` or something, but this trailing semi-colon was causing problems for me.